dev-docs/bidders/prismassp.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: bidder
+title: Prismassp
+description: Prebid Prismassp Bidder Adapter
+pbjs: true
+biddercode: prismassp
+gvl_id: 965
+tcfeu_supported: true
+usp_supported: true
+gpp_supported: true
+schain_supported: true
+dchain_supported: false
+floors_supported: true
+userIds: all
+tcfeu_supported: true
+media_types: banner, video, native
```

Comment:
Can you add the new meta data field `multiformat_supported`. It takes one of the three following values

* will-bid-on-any
* will-bid-on-one
* will-not-bid

See https://docs.prebid.org/formats/multiformat.html for more info.

I know that this is just an alias for the Nexx360 SSP solution. Maybe you could find out and update all the alias configurations